include/zephyr/bluetooth/mesh/dfu.h

```diff
@@ -155,14 +156,10 @@ struct bt_mesh_dfu_slot {
 	size_t fwid_len;
 	/** Length of the metadata. */
 	size_t metadata_len;
-	/** Length of the image URI. */
-	size_t uri_len;
 	/** Firmware ID. */
 	uint8_t fwid[CONFIG_BT_MESH_DFU_FWID_MAXLEN];
 	/** Metadata. */
 	uint8_t metadata[CONFIG_BT_MESH_DFU_METADATA_MAXLEN];
-	/** Image URI. */
```

Comment:
Because the URI does not belong to the image. The image can be transferred in-band, or OOB with different URIs. The URI belongs to the "Upload" operation, not the image itself. (This field in the `bt_mesh_dfu_slot` was previously never used)